cmd/lotus-shed/dealtracker.go

```diff
@@ -0,0 +1,265 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"net"
+	"net/http"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/lotus/api"
+	lcli "github.com/filecoin-project/lotus/cli"
+	"github.com/ipfs/go-cid"
+	"github.com/urfave/cli/v2"
+)
+
+type dealStatsServer struct {
+	api api.FullNode
+}
+
+var filteredClients map[address.Address]bool
+
+func init() {
+	filteredClients = make(map[address.Address]bool)
```

Comment:
Is there a way we can filter out miners as well? E.g. there are terabytes of data from Travis' miners that were stored at genesis on Space Race net, but this isn't legit usage data.

Reply:
So, i'm only looking at deals on chain. Do the genesis miners have deals for their data?

Reply:
Yeah I believe they do have deals on chain!